apps/builder/app/builder/features/settings-panel/variables-section.tsx

```diff
@@ -124,6 +126,20 @@ const $usedVariables = computed(
         }
       }
     }
+    if (page) {
```

Comment:
that list looks extremely suspicious: how are we supposed to know we need to update it if we add more things to page settings?

Reply:
Please don't merge it like this.

Reply:
no way, we have it in pages ui, in compiler, in utils and in variables.

Reply:
Will try to generalize with incremental updates in the future